features/z-index.yml

```diff
@@ -0,0 +1,11 @@
+name: z-index
+description: The `z-index` CSS property allows you to order overlapping elements, with higher values in front of or on top of lower values.
+spec: https://drafts.csswg.org/css2/#z-index
+snapshot: ecmascript-5
```

Comment:
I'm guessing this is just from a mistaken copy and paste.

```suggestion
```

Reply:
This was actually intentional- the [caniuse result](https://caniuse.com/?search=z-index) is for ECMAScript 5, and I made some assumptions about what snapshot was for. In this case, should we omit caniuse completely?

Reply:
That's very strange! I think it came up on caniuse because of some sort of search word splitting. I can't think of any reason `z-index` would be related to ES5.

The `snapshot` field is a bit under-developed (and not published for consumers, at least not yet). It's meant to capture which version of a specification introduced the feature. In this case, `z-index` has no relationship to the ECMAscript 5 spec (as best as I can tell).